test/docs.jl

```diff
@@ -310,6 +312,19 @@ let a = @doc(DocsTest.multidoc),
     @test docstrings_equal(a, b)
 end
 
+let a = Docs.docstrloc(DocsTest.f, Tuple{Any})
+    @test a == [(@__FILE__, DocsTest.F1_LINE_NUMBER)]
+end
+
+let a = Docs.docstrloc(DocsTest.f, Tuple{Any, Any})
+    @test a == [(@__FILE__, DocsTest.F2_LINE_NUMBER)]
+end
+
+let a = Docs.docstrloc(DocsTest.f)
+    @test a == [(@__FILE__, DocsTest.F1_LINE_NUMBER),
+                (@__FILE__, DocsTest.F2_LINE_NUMBER)]
+end
+
```

Comment:
Add a test for the case when there's no docstring attached to a binding?

Reply:
Good idea, I added one.